### PR TITLE
adds mint to mojito recipe

### PR DESCRIPTION
**Pull request by efzapa**
## What Does This PR Do
Adds 1 part mint to the mojito recipe

## Why It's Good For The Game
IRL Mojitos have mint in them, this also makes it 5 parts so you can make an even 100 units in a shaker.

## Testing
just changed a value, _**shouldnt**_ need testing

## Changelog
:cl:
tweak: Making a Mojito now requires 1 part Mint
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/drinks_reactions.dm [hunk 1]
```diff
@@ -164,7 +164,7 @@
 	name = "Mojito"
 	id = "mojito"
 	result = "mojito"
-	required_reagents = list("rum" = 1, "sugar" = 1, "limejuice" = 1, "sodawater" = 1)
+	required_reagents = list("rum" = 1, "sugar" = 1, "limejuice" = 1, "sodawater" = 1 "mint" = 1)
```
Comment (GDNgit) [inline code/modules/reagents/chemistry/recipes/drinks_reactions.dm:167]: You need a , after the "sodawater" = 1

Comment (efzapa) [reply code/modules/reagents/chemistry/recipes/drinks_reactions.dm]: doneso

**Comment by Contrabang** [comment]
As much as I think this is a good idea, I think this means mojitos is almost never going to be made. `"mint"` can only be gotten from the mint botany plant. The chef's mint contains 5u of `"minttoxin"`. And I can't recall having botany grown a mint plant ever

**Comment by Contrabang** [comment]
Thats fair, I just actually remember the last time botany followed through on any of a bartender's requests

**Comment by matttheficus** [comment]
Im eh on this, itll just mean Mojitos will never get made.

**Comment by datlo** [comment]
Suggestion : Add mint candy to some vendor that the bartender can grind to get mint by himself for some cash. Also would let you fix bad breath as other crew 👍

**Comment by efzapa** [comment]
> Suggestion : Add mint candy to some vendor that the bartender can grind to get mint by himself for some cash. Also would let you fix bad breath as other crew 👍

maybe, maybe.
but think of this. what use of mint does mint have outside of mint juleps?
obviously nobody is gonna grow mint if it has no uses.

**Comment by hal9000PR** [comment]
If we hold botany to a higher standard by pushing these types of prs then botany will be more willing to engage with the station

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**